policyengine_us/variables/gov/states/co/tax/income/credits/family_affordability_credit/co_family_affordability_credit.py

```diff
@@ -0,0 +1,32 @@
+from policyengine_us.model_api import *
+
+# FIXME: This is boilerplate taken directly from the reform code.
+
+
+class co_family_affordability_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "Colorado Family Affordability Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://leg.colorado.gov/bills/hb24-1311"
+    defined_for = StateCode.CO
+
+    def formula(person, period, parameters):
+        age = person("age", period)
+        dependent = person("is_child_dependent", period)
+        p = parameters(
+            period
+        ).gov.states.co.tax.income.credits.family_affordability_credit
+        base_amount = p.amount * dependent
+        age_reduction = p.age_reduction.calc(age)
+        agi = person.tax_unit("adjusted_gross_income", period)
+        filing_status = person.tax_unit("filing_status", period)
+        phase_out_start = p.income_reduction.start[filing_status]
+        phase_out_amount = p.income_reduction.rate
```

Comment:
can skip these and use directly in the formula. we generally favor using the `p.` directly for conciseness and to clarify that the statement involves a policy parameter.

also amount means dollar values rather than percentages so please keep as `rate`